README.md

```diff
@@ -1,54 +1,235 @@
+<a name="readme-top"></a>
 
-## Built With:
+<div align="center">
 
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Recipe-App] <a name="about-project"></a>
+
+> A Recipe app that keeps track of all your recipes, ingredients, and inventory. You can save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Recipe-App is built with 
 - Ruby on Rails
-- Postgresql
+- PostgresSQL
+- Bootstrap 
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
 
-To get a local copy up and running follow these simple steps.
+> Describe between 1-3 key features of the application.
 
-## Getting Started
+- **[Shopping list]**
+- **[Food list]**
+- **[Recipe List]**
 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- Install and setup PostgreSQL
+- Install and setup Ruby On Rails 
+- Setup Bootstrap
+
+To get a local copy up and running, follow these steps.
+- git clone https://github.com/ogagaoghene/recipe_app
+- cd recipe_app
 
 ### Prerequisites
-- Ruby
-- Postgresql
-- Rails
 
-## Setup
+In order to run this project you need:
 
-### Clone this repository
-git clone https://github.com/ogagaoghene/recipe_app
+- Ruby on Rails 
+- PostgreSQL
+- Bootstrap 
+- npm install 
+- rails server
 
+### Setup
 
-### Run project
```

Comment:
* I strongly suggest you properly update your readme file setup section on how to properly install and run your app by any visitor of  your repo ( which might be a non tech person ) its not enough for your project to work locally at your end alone, take the visitor step by step on how to achieve setup success as seen at your end 🙏 
* i managed to install it on my end by fixing a lot of migration issues 😓  which might not be the case for a non tech visitor 

> By running `rails db:drop:all` `rails db:create` `rails db:migrate`

> And i removed all duplicate columns ie ( confirmation_token ) which is already existing 
![image](https://user-images.githubusercontent.com/52497246/203884684-ea0a337f-21eb-4eb8-8168-431524a175b2.png)
![image](https://user-images.githubusercontent.com/52497246/203884757-71621e4a-5661-4884-bd2f-0239fcd95d75.png)


![image](https://user-images.githubusercontent.com/52497246/203883695-837a7a50-20bd-4071-9a05-c19029d29120.png)
![image](https://user-images.githubusercontent.com/52497246/203883854-822d3774-0697-4669-bbae-b12b59d4ccde.png)

Reply:
@Davidosky007 I have updated my README file as requested, I am exhausted bro. I anticipate a positive feedback so I can attend to other pressing issues. Thanks in anticipation.